### DISCUSSION TITLE
Default Realm Theme

### DISCUSSION THREAD

**Discussion by naaly17** [keycloak/keycloak#36021] [Q&A]
I previously in Keycloak  v17, to override the default theme for all realms I overwrote the base theme.  Is there a way in v26 to do this? And where is their example themes directory on git? I don't have perms to extract from the jar file.

**Comment by darius-m** [comment]
There are multiple themes (admin, account, etc.), and how to override them differs depending on how they are created (e.g., current account and admin themes use React, so you'll have to set up a builder for them, while others are still based on PatternFly). You can find how to handle either in the [Sever Development guide](https://www.keycloak.org/docs/latest/server_development/index.html#_themes). Regarding overriding the default themes, I don't think it's particularly easy, and it's actively discouraged in the development guide.
